server/docs/hellobooks_api_doc.yaml

```diff
@@ -588,6 +589,33 @@ paths:
               schema:
                 $ref: "#/components/schemas/errorResponse"
 
+  /books/extend/{id}:
+    get:
+      tags:
+        - Books
+      summary: Extend book return date
+      description: Extends the date a book is supposed to be returned
+      parameters:
```

Comment:
You did not add input for the **date**, you added just parameter?

Reply:
Yeah. My bad.